test/cbortest.c

```diff
@@ -146,6 +146,15 @@ static DPS_Status TestTextString()
         ASSERT((len == 0) || (strncmp(examples[i].s, str, len) == 0));
     }
 
+    char bigString[4096];
+    memset(bigString, 'A', 4096);
```

Comment:
String is not NUL terminated but calls a function that expects a NUL terminated string.